sig-testing/README.md

```diff
@@ -47,14 +47,6 @@ Boskos is a resource manager service that handles different kinds of resources a
 - **Owners:**
   - https://raw.githubusercontent.com/kubernetes-sigs/boskos/master/OWNERS
   - https://raw.githubusercontent.com/kubernetes/test-infra/master/boskos/OWNERS
-### gopherage
-Gopherage is a tool for manipulating Go coverage files. We use it on the Kubernetes project to report on code coverage due to e2e tests
```

Comment:
Are we sure we want to drop this?
It is unmaintained but is currently ["used" in CI](https://github.com/kubernetes/test-infra/blob/c68834a35762399bac39eff4bbcef65cb5b9fd12/config/jobs/kubernetes/sig-testing/coverage.yaml#L46) without any planned replacement yet. 
should we wait for a replacement or until is completely unused?
FYI @MushuEE

Reply:
I feel like this is still covered under the test-infra subproject.  There are a variety of codebases in there that are used but in maintenance mode.  If this reaches a level of activity where we're making plans for it, have a team of people rallying around it, or are thinking of extracting it from test-infra, then I think it makes sense to be its own subproject.

Reply:
sgtm